### PR TITLE
Live data

### PR DESCRIPTION
**Pull request by ZoyaStudio**
Implemented use of local storage for user's outfit, completed integration with live data.

### CODE REVIEW

File: Client/src/Components/related-products/ProductCard.jsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
-/* eslint-disable import/extensions */
 import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import axios from "axios";
 import css from "./styles.css";
 import helper from "./helper-functions";
+import StarList from "../product-reviews/star-rating/StarList";
 
 const { getTruePrice } = helper;
```
Comment (alexandercausey) [inline Client/src/Components/related-products/ProductCard.jsx:8]: You could actually move this up to line 5 as 'import { getTruePrice } from "./helper-functions";'.

**Review by alexandercausey — APPROVED** [review]
Looks really good! The local storage looks pretty straight forward too.

File: Client/src/Components/related-products/ProductCard.jsx (hunk 1 above)
Comment (ZoyaStudio) [reply Client/src/Components/related-products/ProductCard.jsx:8]: Webpack will throw an error if I do it all on the same line.

Comment (alexandercausey) [reply Client/src/Components/related-products/ProductCard.jsx:8]: Huh, that's really odd, I was able to destructure right out of the import in a few places... boo webpack

**Pull request merged**